### PR TITLE
Add `bufferedOutput` parameter to adb helpers.

### LINKED ISSUE #11: Emulator.kt should be called AdbDevice.kt
I was trying to find the `AdbDevice` class implementation from Composer, and I couldn't find it on Composer or Commander navigating through GitHub. It took me a (pathetically) long time to find that the class is located in the [Emulator.kt](https://github.com/gojuno/commander/blob/master/android/src/main/kotlin/com/gojuno/commander/android/Emulator.kt) file.

I know it's a small detail, but since it only contains the AdbDevice class which includes both emulators and physical devices, I think it would be best to rename it. That way nobody else will feel as stupid as I did just now 😅 

I'd like to submit the PR, but just wanted to ask first in case I'm missing some reason not to. I think Composer wouldn't be affected because the class is imported with `import com.gojuno.commander.android.*`.

### PR DESCRIPTION
**Pull request by yunikkk**
Let's make it configurable from outside, so we can experiment with different OSes and commands.

Closes #11, too.

### CODE REVIEW

File: android/src/main/kotlin/com/gojuno/commander/android/Adb.kt [hunk 1]
```diff
@@ -22,7 +22,10 @@ private val buildTools: String? by lazy {
 }
 val aapt: String by lazy { buildTools?.let { "$buildTools/aapt" } ?: "" }
 
-fun connectedAdbDevices(): Observable<Set<AdbDevice>> = process(listOf(adb, "devices"), unbufferedOutput = true)
+fun connectedAdbDevices(unbufferedOutput: Boolean = true): Observable<Set<AdbDevice>> = process(
+        listOf(adb, "devices"),
+        unbufferedOutput = unbufferedOutput
```
Comment (artem-zinnatullin) [inline android/src/main/kotlin/com/gojuno/commander/android/Adb.kt:27]: Why should user of adb function care about that though?

Comment (yunikkk) [reply android/src/main/kotlin/com/gojuno/commander/android/Adb.kt:27]: Well, he shouldn't, and the default value is still true.

Comment (yunikkk) [reply android/src/main/kotlin/com/gojuno/commander/android/Adb.kt:27]: Maybe we can use `false` here, too, need to test it more on other OSes. Will check it.

Comment (artem-zinnatullin) [reply android/src/main/kotlin/com/gojuno/commander/android/Adb.kt:27]: ook

**Pull request closed without merging**